tsconfig.json

```diff
@@ -2,11 +2,12 @@
   "extends": "@tsconfig/react-native/tsconfig.json",
   "compilerOptions": {
     "jsx": "react-jsx",
+    "types": ["node"],
```

Comment:
Node?

Reply:
Yeah, we use some node types in places (NodeJS.Timer for instance)